drivers/SmartThings/matter-appliance/profiles/refrigerator-freezer-tn-tl.yml

```diff
@@ -2,37 +2,37 @@ name: refrigerator-freezer-tn-tl
 components:
 - id: main
   capabilities:
-  - id: mode # Refrigerator And Temperature Controlled Cabinet Mode
```

Comment:
Having comments in the profile definition causes issues when parsing this file for capability names. This results in subscription issues because we cannot properly parse the capability name when determining which subscripted attributes we will subscribe to.

cc: @yhoyoon @HunsupJung

Reply:
Yes, It was for implementation. It must be removed.